docs/assets/plugins/metadata/metadata.js

```diff
@@ -392,6 +392,25 @@
         return result.replace(/^ +| +$/gm, '');
       });
 
+      // Handle [component-description] tags
+      content = content.replace(/\[component-description:([a-z-]+)\]/g, (match, tag) => {
```

Comment:
Titles and descriptions are currently shown above and below the header, respectively. While we're in here, it probably makes sense to update the header logic for `[component-header:sl-button]` to show the title/description.

Current:
<img width="479" alt="CleanShot 2022-10-14 at 08 46 06@2x" src="https://user-images.githubusercontent.com/55639/195850398-014830d5-ce0c-46b0-97a6-f6a1f28b7c8c.png">

Proposed:
<img width="256" alt="CleanShot 2022-10-14 at 08 47 07@2x" src="https://user-images.githubusercontent.com/55639/195850464-aa873a13-126c-4dbd-9191-9229fa9c5d2f.png">

But the output would be the same. This might require another JSDoc tag, possibly `@name`.

Reply:
`@name` is not working. Name is always the camelcase TagName

Reply:
Hmm, can we try a custom `@title` tag then? We may need to modify the CEM plugin to look for it if it doesn't include it by default. I can help with that, if need be.

Reply:
Unfortunately, I am reaching my limits. I suspect that changes in "custom-elements-manifest" are necessary for a `@title` tag. But I don't know what they should be 😆

Reply:
Round 2 😃 and happy weekend

Reply:
Sorry for the delay. The latest changes look awesome. Much cleaner, thanks!

Reply:
No problem, I read on Twitter that you have a lot going on. I keep my fingers crossed for you for your further way 👍🏻